### ISSUE TITLE
Welcome dialog GUI issues

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#6350]
Feedback on GUI, received via email.

> There should be a space between the heading 'Welcome to NVDA!' and the remainder of the text under that.
> 
> The following text should come next right under the main welcome heading: 
> Press NVDA+n at any time to activate the NVDA menu. 
> From this menu, you can configure NVDA, get help and access other NVDA functions.
> 
> then a space to seperate out the information a bit 
> Then on to the comments about the different modifiers:
> 
> a few wording changes, nothing major. 
> By default, the numpad insert and main insert keys may both be used 
> may both should be are both... as in this should be 
> By default, the numpad insert and main insert keys are both used
> 
> You can also configure NVDA to use the CapsLock as the NVDA key. 
> the should be an, because it is one of many, so it should be: 
> You can also configure NVDA to use the CapsLock as an NVDA key.
> 
> then a space again, and the options
> 
> Automatically start NVDA after I log on to Windows 
> should be: 
> Automatically start NVDA after logging on to Windows 
> (this is just to be consistent with the fact that we do not use the first person in any of our dialogs)
> 
> Show this dialog when NVDA starts 
> should be 
> Always show this dialog when NVDA starts
> 
> and I think there should be an OK/Cancel group instead of just OK, and then this group should be right aligned for consistency with other dialogs
